examples/nidaqmx/analog-output.py

```diff
@@ -50,49 +50,49 @@
 task = None
 
 
-def raise_if_error(response):
-    """Raise an exception if an error was returned."""
-    if response.status != 0:
-        response = client.GetErrorString(
-            nidaqmx_types.GetErrorStringRequest(error_code=response.status)
-        )
-        raise Exception(f"Error: {response.error_string}")
-
-
 try:
+    def check_for_warning(response):
```

Comment:
In other examples we define the `check_for_warning` method above/before the try block. Might want to do the same for DAQ if possible.